temporalcli/commands.schedule.go

```diff
@@ -0,0 +1,484 @@
+package temporalcli
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/temporalio/cli/temporalcli/internal/printer"
+	enumspb "go.temporal.io/api/enums/v1"
+	schedpb "go.temporal.io/api/schedule/v1"
+	"go.temporal.io/api/workflowservice/v1"
+	"go.temporal.io/sdk/client"
+	"go.temporal.io/server/common/primitives/timestamp"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+type printableSchedule struct {
+	ScheduleId       any
+	CalendarSpecs    []any `cli:",cardOmitEmpty"`
+	IntervalSpecs    []any `cli:",cardOmitEmpty"`
+	WorkflowId       any
+	WorkflowType     any
+	Paused           any
+	Notes            any `cli:",cardOmitEmpty"`
+	NextRunTime      any
+	LastRunTime      any
+	RunningWorkflows []any
+	SearchAttributes any `cli:",cardOmitEmpty"`
+	Memo             any `cli:",cardOmitEmpty"`
```

Comment:
Not required, but I figure at least some of these could have stronger field types

Reply:
Big +1 here